events/wc_quest_events.txt

```diff
@@ -888,6 +882,12 @@ society_quest_event = {
 
 		ai_chance = { factor = 0 }
 	}
+
+	after = {
+		clr_character_flag = control_mind_martial_flag
+		clr_character_flag = control_mind_intrigue_flag
+		clr_character_flag = control_mind_learning_flag
+	}
```

Comment:
I would use `clear_flags_with_prefix`, but that's ok too

Reply:
There may be other flags in the future starting with `control_mind_` or even `quest_control_mind_`. I wouldn't do it.